packages/blog-starter-kit/themes/enterprise/components/analytics.tsx

```diff
@@ -138,5 +117,91 @@ export const Analytics = () => {
 		});
 	};
 
+	function _sendViewsToAdvancedAnalyticsDashboard() {
+		const publicationId = publication.id;
+		const postId = post && post.id;
+		const seriesId = series?.id || post?.series?.id;
+		const staticPageId = page && page.id;
```

Comment:
`series` and `page` are not defined. They have to be passed as a prop to this `Analytics` component.

Check [this commit in our starter kit](https://github.com/Hashnode/starter-kit/commit/f543e14caf8cf017416638d8d60a3e1e3a0c1de1#diff-049e1536cc838d860847f370d4507d0a86e4cb72ba91887741f73d3d4ffcc3ff) that made the change.

Reply:
I updated the analytics.tsx file to be exactly as in the commit you linked.

Reply:
There are some changes missing outside of this file.

I have created a PR with the missing required changes. You can either merge it or do the changes on this branch if you want.

https://github.com/mindsdb/hashnode-starter-kit/pull/19/files

Reply:
Thanks @JannikWempe 
I'll close this PR and merge the one you created.

Reply:
You are welcome 😊
You can tag me if you have a question or need help.